docs/Guides/visualizing-mcap-in-foxglove-studio/visualize-in-foxglove-studio.md

```diff
@@ -0,0 +1,23 @@
+# Visualize in Foxglove Studio
+
+## Using Foxglove schemas
```

Comment:
similar to the previous comment, I wonder if mcap repo is the place for this, or whether we could just give a link here to some docs on the studio side to describe this stuff. We will presumably need to duplicate this info in studio docs.